var/spack/repos/builtin/packages/intel-mkl/package.py

```diff
@@ -83,6 +83,34 @@ def blas_libs(self):
     def lapack_libs(self):
         return self.blas_libs
 
+    @property
+    def scalapack_libs(self):
+        libnames = [ 'libmkl_scalapack' ]
+        if self.spec.satisfies('^openmpi'):
+            libnames.append('libmkl_blacs_openmpi')
+        elif self.spec.satisfies('^mpich@1'):
+            libnames.append('libmkl_blacs')
+        elif self.spec.satisfies('^mpich@2:'):
+            libnames.append('libmkl_blacs_intelmpi')
```

Comment:
I think u r right. My notes were probably incorrect. I also checked link advisor with GCC and OpenMPI, Mpich2 and there is no blacs_mpich. Sorry for confusion.